### PR TITLE
Add more constructors to SubscribeCommandParameters

### PR DESCRIPTION
**Pull request by kblok**
New constructors on the SubscribeCommandParameters class.

### CODE REVIEW

File: src/WebDriverBiDi/Session/SubscribeCommandParameters.cs [hunk 1]
```diff
@@ -23,6 +23,28 @@ public SubscribeCommandParameters()
     {
     }
 
+    /// <summary>
+    /// Initializes a new instance of the <see cref="SubscribeCommandParameters"/> class.
+    /// </summary>
+    /// <param name="events">The list of events to which to subscribe or unsubscribe.</param>
+    /// <param name="contexts">The list of browsing context IDs for which to subscribe to or unsubscribe from the specified events.</param>
+    public SubscribeCommandParameters(string[] events, string[] contexts)
```
Comment (jimevans) [inline src/WebDriverBiDi/Session/SubscribeCommandParameters.cs:31]: Would one constructor using `IList<string>` args work better? That would cover both `List<string>` and `string[]`, I think (I may be wrong).

**Review by jimevans — CHANGES_REQUESTED** [review]
Add unit tests to cover the new constructors?

File: src/WebDriverBiDi/Session/SubscribeCommandParameters.cs (hunk 1 above)
Comment (kblok) [reply src/WebDriverBiDi/Session/SubscribeCommandParameters.cs:31]: I think that can work. 👍

Comment (kblok) [reply src/WebDriverBiDi/Session/SubscribeCommandParameters.cs]: Done!

**Review by jimevans — APPROVED** [review]

**Pull request merged**